apps/rush-lib/src/utilities/Utilities.ts

```diff
@@ -357,37 +409,19 @@ export class Utilities {
   /**
    * Executes the command using cmd if running on windows, or using sh if running on a non-windows OS.
    * @param command - the command to run on shell
-   * @param workingDirectory - working directory for running this command
-   * @param initCwd = the folder containing a local .npmrc, which will be used
-   *        for the INIT_CWD environment variable
-   * @param captureOutput - if true, map stdio to 'pipe' instead of the parent process's streams
+   * @param options - options for how the command should be run
    */
   public static executeLifecycleCommandAsync(
     command: string,
-    workingDirectory: string,
-    initCwd: string,
-    captureOutput: boolean = false
+    rushConfiguration: RushConfiguration,
```

Comment:
Why isn't `rushConfiguration` part of `ILifecycleCommandOptions`?  I don't see anything special about it.